bin/module-install

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+try {
+  const opt = require('opt-cli')
+  if (!opt.testOptOut('ghooks-install')) {
+    require('./install')
+  }
+} catch (e) {
+  console.warn( // eslint-disable-line no-console
+    '\nghooks postinstall could not be completed.\n' +
+    '==========================================\n\n' +
+    'Usually this is due to using npm@2.15.x. Please consider updating npm\n' +
+    'to its latest version or run the following command manually afterwards:\n\n' +
+    '\tnode ./bin/install\n'
```

Comment:
Maybe this should use [`npm explore`](https://docs.npmjs.com/cli/explore):

```
npm explore ghooks -- node ./bin/install
```

Just tested that locally and it seems to work :-)